framework/utils/mathUtils.py

```diff
@@ -513,19 +513,31 @@ def NDInArray(findIn,val,tol=1e-12):
     return False,None,None
   return found,idx,looking
 
-def numBinsDraconis(data):
+def numBinsDraconis(data, low=None, alternateOkay=True):
```

Comment:
you may provide a default value for low if you think some number is good for general cases?

Reply:
I think it would be a mistake for the generic method to have a default. While for the ARMA cases we've seen 20 is a good value, that will almost certainly not be true in general.